Source/KIS_Shared.cs

```diff
@@ -1363,7 +1363,13 @@ public enum MessageAction {
     Vessel newVessel = newPart.gameObject.AddComponent<Vessel>();
     newVessel.id = Guid.NewGuid();
     if (newVessel.Initialize(false)) {
-      newVessel.vesselName = newPart.partInfo.title;
+      var item = newPart.FindModuleImplementing <ModuleKISItem> ();
+      if (item != null && !item.vesselAutoRename) {
```

Comment:
This condition doesn't keep the default behavior. In case of there is no module `ModuleKISItem`, then new naming mechanism will be used.

Reply:
oops, inverted logic.